packages/apollo-server-cloudflare/src/ApolloServer.ts

```diff
@@ -1,10 +1,16 @@
 import { graphqlCloudflare } from './cloudflareApollo';
 
 import { ApolloServerBase } from 'apollo-server-core';
+export { GraphQLOptions, GraphQLExtension } from 'apollo-server-core';
+import { GraphQLOptions } from 'apollo-server-core';
 
 export class ApolloServer extends ApolloServerBase {
+  async createGraphQLServerOptions(req: Request): Promise<GraphQLOptions> {
```

Comment:
Why does this function take a request argument rather than an options argument like the one it calls? Also what is the point of this function and why does it have a different name than graphQLServerOptions?